seisbench/models/base.py

```diff
@@ -213,6 +344,23 @@ def _parse_metadata(self):
         default_args = self._weights_metadata.get("default_args", {})
         self.default_args.update(default_args)
 
+    def _get_input_args(self, obj):
+        signature = inspect.signature(obj)
+        args = {k: v._default for k, v in signature.parameters.items()}
+        if "kwargs" in args.keys():
+            del args["kwargs"]
+        return args
+
+    @abstractmethod
+    def get_model_args(self):
```

Comment:
I think this should return `{"citation": self.citation}`

Reply:
Correct, missed the top level out and is now included and passed down the object hierarchy.